src/views/Home.jsx

```diff
@@ -1,12 +1,52 @@
 import './Home.css';
 import { SingleList } from '../components/SingleList';
+import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { createList } from '../api/firebase.js';
+import { useAuth } from '../api';
 
 export function Home({ data, setListPath }) {
+	const { user } = useAuth();
+	const userId = user?.uid;
+	const userEmail = user?.email;
+	const navigate = useNavigate();
+
+	const [listName, setListName] = useState('');
+
+	const handleInputChange = (event) => {
+		setListName(event.target.value);
+	};
+
+	const handleSubmit = async (event) => {
+		event.preventDefault();
+		if (listName) {
+			const path = await createList(userId, userEmail, listName);
+			setListPath(path);
+			setListName('');
+			navigate('/list');
+			alert(`New list was created named: ${listName}`);
+		} else {
+			alert("Error occurred. List wasn't created");
+		}
+	};
+
 	return (
 		<div className="Home">
 			<p>
 				Hello from the home (<code>/</code>) page!
 			</p>
+			<form id="item-form" onSubmit={handleSubmit}>
+				<label htmlFor="list-name">Enter a new list name:</label>
+				<input
+					type="text"
+					name="list-name"
+					id="list-name"
+					value={listName}
+					onChange={handleInputChange}
+					required
```

Comment:
Not something that needs to happen right away, but we probably should not rely on the default 'alert' and HTML built-in validations long term. 

Quote from ej, another TCL mentor who is brilliant about accessibility: 

> The visual design of the default validation is hard to customize, so it won't look like the rest of your app, and tragically it is also inaccessible.

I will share another post in our Slack channel about this.